sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/implementation/utils/AccessTokenUtil.java

```diff
@@ -121,14 +157,25 @@ public static String getAccessToken(String resource, String aadAuthenticationUrl
      * @return the authorization token.
      */
     private static String getAccessTokenOnAppService(String resource, String clientId) {
+        OAuthToken result;
+        result = getAccTokenOnAppService(resource, clientId);
+        return result == null ? null : result.getAccessToken();
+    }
+
+    /**
+     * Get the access token on Azure App Service.
+     *
+     * @param resource the resource.
+     * @param clientId the user-assigned managed identity (null if system-assigned).
+     * @return the authorization token.
+     */
+    private static OAuthToken getAccTokenOnAppService(String resource, String clientId) {
         LOGGER.entering("AccessTokenUtil", "getAccessTokenOnAppService", resource);
```

Comment:
the log's information is not same to this method name. I suggest change the method name.